instrumentation/rack/opentelemetry-instrumentation-rack.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-common', '~> 0.19.3'
   spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.21.0'
+  spec.add_dependency 'opentelemetry-sdk-experimental'
```

Comment:
I wonder if we should include some kind of note during upgrade to next release about how `sdk-experimental` won't replace the SDK. Just don't want to freak users out.

Reply:
Actually, since making this optional, it occurs to me that we don't need to add the experimental SDK to the dependencies since it's up to the user to inject the `ResponseTextMapPropagator` in the middleware config.

Reply:
You're right 🧠! Hot dog!